lib/dnd.js

```diff
@@ -85,6 +83,8 @@ Dnd.prototype._createTraveler = function (d, i, node) {
 
   this.tree.el.node().appendChild(this.traveler.node())
 
+  this.tree.emit('dndstart', node, this.traveler.node(), d)
```

Comment:
At the moment there's no documentation for this anywhere, so really this method call is what serves as the documentation at this point.  Would you mind at least putting a comment to indicate what `d` is?  I can't tell what it is.